detox/e2e/support/ui/component/channel_sidebar.js

```diff
@@ -18,6 +18,8 @@ class ChannelSidebar {
 
     toBeVisible = async () => {
         await expect(element(by.id(this.testID.mainSidebar))).toBeVisible();
+
+        return this.mainSidebar;
```

Comment:
We should rename the file to be `main_sidebar` as well as rename the class name. 1/5

Reply:
I was actually thinking of renaming all main sidebar to channel sidebar. I feel channel sidebar sounds more appropriate and more in line with settings sidebar (given that we have channel drawer button and settings drawer button). But I didn't do it because you guys might have a reason for naming it as main sidebar.

I'm ok either way. Let me know if you have preference and I can rename the files and variables accordingly.

Reply:
Actually main sidebar makes more sense as currently you have channels and teams & for v2 you'll have channes / teams & servers.

Reply:
we could consider renaming channel_drawer_button to be main_sidebar_button and settings_sidebar_button

Reply:
Updated